knowledge-base/how-to-change-report-server-iis-user.md

```diff
@@ -0,0 +1,70 @@
+---
+title: How to Create a limited IIS User for Report Server and Service Agent
+description: this tutorial shows how to use a lower privlidged user to host Report Server in IIS
+type: how-to
+page_title: How to change IIS user for Report Server and Service Agent
+slug: how-to-change-report-server-iis-user
+tags: report server, iis, user permissions, iis permissions
+res_type: kb
+---
+
+## Environment
+
+|  |  |
+|---------|-----------------|
+| Product | Progress® Telerik® Report Server |
+| Version | 10.1.24.514 |
+
+## Description
+
+This tutorial will show you how to change the Report Server's IIS application pool and service agent to use an identity with limited permissions.
+
+## Solution
+
+1. Create a new **Windows** User with limited permissions.
```

Comment:
Maybe we should add a short sentence like 'For this tutorial we chose to name it RSUser'? This way, when the people see 'RSUser' on step 2, they won't think it MUST be named 'RSUser'?
I understand it's pretty self-explanatory, but AFAIK the target audience of this document is not strictly devs and admins.

Reply:
done. Now that there are three additional comments for this step, I moved them all into a bullet list under the step.